recipes/dcmtk/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "DCMTK" %}
+{% set version = "3.6.7" %}
+
+package:
+  name: {{ name | lower }}
+  version: {{ version }}
+
+source:
+  url: https://dicom.offis.de/download/dcmtk/dcmtk{{ version | replace('.','') }}/dcmtk-{{ version }}.tar.gz
+  sha256: 7c58298e3e8d60232ee6fc8408cfadd14463cc11a3c4ca4c59af5988c7e9710a
+
+build:
+  number: 0
```

```suggestion
build:
  number: 0
  run_exports:
    - {{ pin_subpackage( name|lower, max_pin='x.x.x' ) }}
```
Need a run export if you're shipping libs and headers. Pin according to ABI guarantees.

---

Done in 45f31d6b03, pinned to `major.minor`.

---

You need to include the keyword `max_pin` because `max_pin` is not the second positional argument for this function. i.e.

```
    - {{ pin_subpackage( name|lower, max_pin='x.x' ) }}
```